smoke-tests/src/test/resources/otel.yaml

```diff
@@ -29,10 +31,10 @@ service:
     traces:
       receivers: [ otlp, zipkin, jaeger ]
       processors: [ batch ]
-      exporters: [ logging, otlp ]
+      exporters: [ logging/logging_debug, otlp ]
```

Comment:
just checking that you intend this change (no objection to it)

Reply:
Yes, what Mateusz said :)